### DISCUSSION TITLE
Is there a way to auto delete objects?

### DISCUSSION THREAD

**Discussion by Saqib92** [fabricjs/fabric.js#8594] [Q&A]
I was wondering that is there anything we can do for objects to get deleted automatically??
something like:

`autoDeleteTimeOut: 5000` // gets deleted after 5 seconds. 

or is there any other way we can delete without using `setInterval()`? it will cause performance issue if there are many objects on canvas any looping through each on every second? 
Actually i want to have `timeout` feature in my app.

**Comment by melchiar** [comment]
You can do this with `setTimeout()`. If you're dealing with so many objects that performance is an issue then try grouping the objects to be deleted.
